app/main/views/direct_award.py

```diff
@@ -20,6 +20,11 @@ def list_projects():
     page = get_valid_page_or_1()
 
     projects = DirectAwardProject.query
+
+    with_users = convert_to_boolean(request.args.get('with-users', False))
+    if not isinstance(with_users, bool):
+        abort(400, "with-users param must be True of False")
```

Comment:
Are we really saying that `convert_to_boolean` doesn't guarantee a bool output??

Reply:
Yep. If it gets a thing that it can't convert to boolean (like a random string) it just returns it.